### ISSUE TITLE
How to find a direct chat room with a room info?

### ISSUE THREAD

**Issue by adantenr** [matrix-org/matrix-js-sdk#1060]
I know that a room is classified to PEOPLE in a riot app when it is tagged with direct chat. How can I find a room is tagged with direct chat in code, specially in the Room object?

**Comment by t3chguy** [comment]
You must look at the `m.direct` account data event, react-sdk has a util class for keeping track of DM rooms, this will probably get much simpler once Canonical DMs lands and there can be at most 1 canonical DM between two distinct users: https://github.com/matrix-org/matrix-react-sdk/blob/41792a23a60de87a69024037c7128b5fb91b0b3c/src/utils/DMRoomMap.js

**Comment by adantenr** [comment]
@t3chguy Thank you for the information.

I found a DM room id list in the `m.direct` event,
```
const dmEvent = client.getAccountData('m.direct');
```
and used this information to make a DM room list by comparing room ids.
```
if (dmRoomIdList.includes(room.roomId)) {
            peopleList.push(room);
}
```

FYI, Favorites and Low Priority rooms can be identified by the `tags` property in the `Room` object.
```
if ('m.favourite' in room.tags) {
    favoritesList.push(room);
}

if ('m.lowpriority' in room.tags) {
    lowpriorityList.push(room);
}
```
